task/04-arrays-tasks.js

```diff
@@ -200,7 +203,8 @@ function getTail(arr, n) {
  *    +'30,31,32,33,34'
  */
 function toCsvText(arr) {
-   throw new Error('Not implemented');
+   let newArr = arr.map(a => a.join(','));
+   return newArr.join('\n');
 }
```

Comment:
using bad variable names that doesn't have any hint on what's stored in them
if you can do without declaring variables, it is better to immediately return the value you need, i.e `return arr.map(..).join(..)`